jenkins-design-language/less/components/jtable.less

```diff
@@ -105,10 +105,20 @@ a.JTable-row--href {
 }
 
 .JTable-cell-contents {
-    width:100%;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 100%;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+
+    > code,
+    .text-with-ellipsis-container {
```

Comment:
We are repeating this sequence of CSS rules in a bunch of places:

```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```

Perhaps we should make this a discrete and more generic className like `.ellipsis` - the name is brief, but I think the semantics are clear. Then we can apply it where needed, and also use it as a mixin in the other less rules so they are a bit more concise.